src/test/java/org/numenta/nupic/network/LayerTest.java

```diff
@@ -41,13 +41,12 @@
 import org.junit.Test;
 import org.numenta.nupic.Parameters;
 import org.numenta.nupic.Parameters.KEY;
-import org.numenta.nupic.algorithms.Anomaly;
+import org.numenta.nupic.algorithms.*;
```

Comment:
@Hopding  I'm not sure if this view changes when you update after I comment, but it looks to me like this wasn't changed to be an explicit import?